ProcessLib/HT/StaggeredHTFEM-impl.h

```diff
@@ -87,45 +93,44 @@ void StaggeredHTFEM<ShapeFunction, IntegrationMethod, GlobalDim>::
         auto const& dNdx = ip_data.dNdx;
         auto const& w = ip_data.integration_weight;
 
-        double p_at_xi = 0.;
-        NumLib::shapeFunctionInterpolate(local_p, N, p_at_xi);
-        double T1_at_xi = 0.;
-        NumLib::shapeFunctionInterpolate(local_T1, N, T1_at_xi);
+        double p_int_pt = 0.0;
+        double T1_int_pt = 0.0;
+        NumLib::shapeFunctionInterpolate(local_p, N, p_int_pt);
+        NumLib::shapeFunctionInterpolate(local_T1, N, T1_int_pt);
```

Comment:
I think you can do the interpolation for both quantities at once
```c++
NumLib::shapeFunctionInterpolate(local_x, N, T1_int_pt, p_int_pt);
```
Order of variables `x_int_pt` must be the same as in `local_x`